src/main/java/com/conveyal/datatools/common/status/MonitorableJob.java

```diff
@@ -288,15 +288,16 @@ private String getCallingMethodTrace() {
          * Shorthand method to update status object on successful job completion.
          */
         public void completeSuccessfully(String message) {
-            this.complete(false, message);
+            // Do not overwrite the message (and other fields), if the job has already been completed.
+            if (!this.completed) this.complete(false, message);
```

Comment:
It kind of seems like an error to call completeSuccessfully twice. If anything, I'd throw an IllegalStateException if the status is already complete. What is calling this twice anyways?

Reply:
Is this still an issue, @evansiroky? This was introduced in https://github.com/ibi-group/datatools-server/pull/371/commits/a692f9c9eef3966fe2925e13df75c9c7f7331731 by @landonreed to address the double call you are referring to.

Reply:
Yep, I think we can resolve this.